src/mindMapPreviewWebview.ts

```diff
@@ -118,11 +125,15 @@ class MindMapPreview {
         fs.writeFileSync(tempImage, html);
     }
 
-    updatePreview() {
-        const data = this.editingEditor.document.getText();
-        this.view.webview.postMessage({
-            "command": "renderMarkdown", "data": data
-        });
+    updatePreview(conf = { fromFile: false }) {

```

Comment:
You are making the code complex. Just simply write like this:
```suggestion
    updatePreview(isFormFile: bool = false) {
        const data = isFromFile ?
            utils.getMarkDownTitle(data) : this.editingEditor.document.getText();
        this.view.webview.postMessage({ command: "renderMarkdown", data: data });
}
```

If you have to use the `conf` variable, please describe why.

Reply:
为了方便扩展参数，语义上更清晰：
1. updatePreview({ isFormFile: true, otherFlag: [] }); 有 Key 名，能够知道添加参数的意义；
2. 个人觉得比这种形式好： updatePreview(true, []);

Reply:
1. 个人认为没有必要过早优化，使用单个参数已经足够，若今后需要扩展，可以再重构。
 2. 注意到，该函数是一个类的方法，故若能从实例中取值，则不应以方法取值。
     ```suggestion
         updatePreview() {
             const data = this.isFromFile ? utils.getMarkDownTitle(data)
                                          : this.editingEditor.document.getText();
         }
     ```
 3. 目前来看使用三元表达式是可取的，惟需再在增加更多判断条件时重构。